libs/cardano-ledger-test/bench/Bench/Cardano/Ledger/ApplyTx.hs

```diff
@@ -177,8 +182,8 @@ applyTxBenchmarks =
         "Deserialise Shelley Tx"
         [ deserialiseTxEra (Proxy @ShelleyBench),
           deserialiseTxEra (Proxy @AllegraBench),
-          deserialiseTxEra (Proxy @MaryBench),
-          deserialiseTxEra (Proxy @AlonzoBench)
+          deserialiseTxEra (Proxy @MaryBench)
+          --deserialiseTxEra (Proxy @AlonzoBench)
```

Comment:
why are we disabling the tests for Alonzo?

Reply:
It's not a test, but a benchmark, we can afford to disable a benchmark. Which is important :smile: 

It has to do with problems with generator for `Alonzo`. See a few lines above `benchApplyTx` is also disabled for `Alonzo`. It looks like it will take more effort to get it implemented for Alonzo.